src/backends/onnx.js

```diff
@@ -59,11 +59,10 @@ const supportedDevices = [];
 /** @type {ONNXExecutionProviders[]} */
 let defaultDevices;
 let ONNX;
-const ORT_SYMBOL = Symbol.for('onnxruntime');
 
-if (ORT_SYMBOL in globalThis) {
-  // If the JS runtime exposes their own ONNX runtime, use it
-  ONNX = globalThis[ORT_SYMBOL];
+if (apis.IS_EXPOSED_RUNTIME_ENV) {
+    // If the JS runtime exposes their own ONNX runtime, use it
+    ONNX = globalThis[apis.EXPOSED_RUNTIME_SYMBOL];
```

Comment:
Could we maybe add some checks here to ensure that the runtime implements the necessary functions - throwing an error otherwise? May not be necessary, but just in case `onnxruntime` maps to something else in `globalThis` (and not a valid runtime)

Reply:
Sure, it makes sense.

Reply:
Hey Joshua, I did add it on https://github.com/huggingface/transformers.js/pull/976/commits/f8da67af8c09333dbc59d3c5563a514b57268da6